### ISSUE TITLE
Regression: small jump when scrolling in the list of operations

### ISSUE THREAD

**Issue by bnjbvr** [bnjbvr/kresus#430]
I was developing on the `minefield` branch and remarked that if you have enough operations, in the main screen (operations list), if you scroll down quickly enough, then you can observe a very small reflow after the scroll. Like a very very small jump once you're done scrolling. See video.

![jumps](https://cloud.githubusercontent.com/assets/1180019/16672382/17382efa-44a6-11e6-9982-2b8e5310c132.gif)

@ZeHiro, can you figure out what's causing this? It triggers under Firefox and Chrome, so at this point this has to be a bug on our side. Any idea?

**Comment by bnjbvr** [comment]
Oh, forgot to add: I've bisected this, this regression was introduced by this change:

commit cfeb79c2d8e614f03618cabbe9cc4d0b76daf3e4
Author: ZeHiro <>
Date:   Fri Jun 17 01:40:14 2016 +0200

    Improve mobile display + allow overflow of types and categories labels

**Comment by bnjbvr** [comment]
This fixes it, fwiw:

```diff
--- a/client/css/style.css
+++ b/client/css/style.css
@@ -402,7 +402,7 @@ ul {
     }
 }
 
-.operation-panel .table td > button, .operation-panel .table td > div > span,
+.operation-panel .table td > div > span,
 .operation-panel .table td > div > div > span
 {
     position: absolute;
```

**Comment by bnjbvr** [comment]
Ha, probably this rules conflicts with the one just below:
https://github.com/bnjbvr/kresus/blob/incoming/client/css/style.css#L405
https://github.com/bnjbvr/kresus/blob/incoming/client/css/style.css#L413

Button which appear on screen are both affected by the first rule and the second rule, which dictate different widths...

**Comment by bnjbvr** [comment]
I'm considering applying the fix mentioned in https://github.com/bnjbvr/kresus/issues/430#issuecomment-231234356 . Does it look fine to you @ZeHiro ?

**Comment by ZeHiro** [comment]
@bnjbvr The two rulez are not conflicting they are complementary. The first one is for standard buttons, the other one is for buttons with bills. They shouldn't be conflicting.

**Comment by ZeHiro** [comment]
Hard coded height might not be a good idea.

**Comment by bnjbvr** [comment]
Good catch! Indeed this fixes the small jump issue I'm seeing (fwiw, I can see it on different machines).

Hard coding a value is not a bad idea: hard coding the same value at different places is a very a bad one. But I am not sure what else we can do...

**Comment by bnjbvr** [comment]
Fixed by  2d80601.
